tests/scripts/thread-cert/thread_cert.py

```diff
@@ -529,48 +570,59 @@ def _prepare_backbone_network(self):
         """
         Prepares one or multiple backbone network(s).
 
-        This method creates the backbone network based on the `backbone` values defined in the TOPOLOGY in each test.
-        If no backbone values are defined, it sets the default backbone network as BACKBONE_DOCKER_NETWORK_NAME
-        from the config module.
+        1. If there's `backbone_network` defined in the TOPOLOGY, it creates the backbone network based on the value.
+           The format of the
+             * backbone name is `backbone{PORT_OFFSET}.{backbone_network}`, for example, "backbone0.0", "backbone0.1";
+             * backbone prefix is `backbone{PORT_OFFSET}:{backbone_network}`, for example, "9100:0::/64", "9100:1::/64".
+
+        2. If no `backbone_network` is defined, it creates the default backbone network, the format of the
+             * backbone name is `backbone{PORT_OFFSET}`, for example, "backbone0";
+             * backbone prefix is `backbone{PORT_OFFSET}::/64`, for example, "9100::/64".
         """
-        # Use backbone_set to store all the backbone values in TOPOLOGY
+        # Create backbone_set to store all the (backbone_name, backbone_prefix) pairs by parsing TOPOLOGY.
         backbone_set = set()
         for node in self.TOPOLOGY:
-            backbone = self.TOPOLOGY[node].get('backbone_network')
-            if backbone:
-                backbone_set.add(backbone)
+            backbone_num = self.TOPOLOGY[node].get('backbone_network')
+            if backbone_num is not None:
+                backbone_set.add((f'{config.BACKBONE_DOCKER_NETWORK_NAME}.{backbone_num}',
+                                  f'{config.BACKBONE_IPV6_ADDR_START}:{backbone_num}::/64'))
```

Comment:
return early.
```suggestion
            backbone_num = self.TOPOLOGY[node].get('backbone_network')
            if backbone_num is None:
                continue
            backbone_network_name = # ... code to compose network name
            if backbone_network_name in backbone_set:
                continue
            backbone_set.add(backbone_network_name)
            # ... code to set the prefix.
```

Reply:
I prefer not modifying this. 

looks like `backbone_set` only stores backbone_network_name with this change, then where to set the prefix?

Reply:
@zesonzhang Should you use a dict instead?

Reply:
@superwhd agree that a dict can make it to more clear and more safe to get the value. But considering `backbone_set ` is only used inside this function and only two elements are stored inside the tuple. it should be fine to store tuples inside the `backbone_set`. Any other concerns?

Reply:
Let's continue discussion in the new comment which suggests to use a set to cover the IDs.